specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/Common.json

```diff
@@ -290,16 +296,73 @@
           "type": "boolean",
           "description": "Whether to trust the server certificate",
           "default": false
+        },
+        "platform": {
+          "$ref": "#/definitions/SqlServerSourcePlatform",
+          "description":  "Server platform type for connection"
         }
       },
-      "required": [ "dataSource" ]
+      "required": [
+        "dataSource"
+      ]
+    },
+    "SqlServerSourcePlatform": {
+      "type": "string",
+      "description": "An enumeration of source platform types",
+      "enum": [
+        "SqlOnPrem"
+      ],
+      "x-ms-enum": {
+        "name": "SqlSourcePlatform",
+        "modelAsString":  true
+      }
+    },
+    "MySqlConnectionInfo": {
+      "x-ms-discriminator-value": "MySqlConnectionInfo",
+      "type": "object",
+      "description": "Information for connecting to MySQL source",
```

Comment:
Should this be MySQL server? As it is used for both source and target

Reply:
Done.